### ISSUE TITLE
Melt rate spatial distribution for Ocean0 

### ISSUE THREAD

**Issue by claireyung** [misomip/isomip-plus#3]
All experiments? Is this a useful plot - they are all quite similar?

**Comment by xylar** [comment]
> All experiments?

Are you referring to all 12 model submissions?  If so, I think it is useful to show them all even if they look a bit similar.  I would not show more than one configuration (e.g. from COCO, which submitted on for each Gamma_T value).

**Comment by claireyung** [comment]
Yep - just the 12 different models (or models run with different coordinates/groups) with their final tuning parameters for Ocean0 COM. Could relate the spatial distributions of melt to how discrete the vertical coordinate makes the ice shape, or the [overturning/barotropic circulation](https://github.com/misomip/isomip-plus/issues/6)

**Comment by xylar** [comment]
> 12 different models (or models run with different coordinates/groups)

We will presumably need to come up with a concise term for this in the paper.  Tricky!

**Comment by claireyung** [comment]
Models are not stable at the end of a year, so we decided to change this plot to be the last year of Ocean1, which is also in a quasi-steady warm state.

**Comment by xylar** [comment]
(Mostly) addressed by #19.

**Comment by claireyung** [comment]
updated plots:

![image](https://github.com/misomip/isomip-plus/assets/61528379/f072a26a-bda1-41e0-b993-92f3860dd0fc)

Also for Ocean2 in https://github.com/misomip/isomip-plus/pull/21

![image](https://github.com/misomip/isomip-plus/assets/61528379/d9c653c5-9175-4655-a2dd-fde626aed496)
